digitiser-aggregator/src/frame/cache.rs

```diff
@@ -33,11 +33,11 @@ where
         digitiser_id: DigitizerId,
         metadata: &FrameMetadata,
         data: D,
-    ) {
+    ) -> bool {
```

Comment:
To signify success or failure with no additional information prefer using `Result<(), ()>`.